core/controllers/topic_editor.py

```diff
@@ -132,7 +133,6 @@ def get(self, topic_id):
             'INTERACTION_SPECS': interaction_registry.Registry.get_all_specs(),
             'interaction_templates': jinja2.utils.Markup(
                 interaction_templates),
-            'dependencies_html': jinja2.utils.Markup(dependencies_html)
```

Comment:
We can remove this here because we only can use interactions without dependencies? Ditto for the other pages.

Reply:
No, one dependency is used here ui codemirror.
Check line 104.

Reply:
However, it has been removed since the dependencies are now added via webpack